### PR TITLE
Scroll to active animal on selection

### LINKED ISSUE #9: Feature: Scroll ambassadors list to active ambassador
It would be nice to scroll the list to the active ambassador when chosen by chat. I can send a pull request if you want.

I would probably would render some attribute (`data-ambassador-id={animal.name}`) to each button and then use something like:
```typescript
const top = animalList.current.querySelector(`[data-ambassador-id="${activeAnimal.name}"]`).offsetTop;
animalList.current.scrollTo({top, behavior: 'smooth'});
```
in the effect.

### PR DESCRIPTION
**Pull request by pjeweb**
Resolves #9

Tested in current Chrome, FF and Safari

### CODE REVIEW

File: src/pages/overlay/components/AmbassadorList.tsx [hunk 1]
```diff
@@ -11,10 +11,29 @@ import AnimalButton from '../../../utils/global/animalButton/AnimalButton'
 import styles from './ambassadorList.module.css'
 import arrow from '../../../assets/arrow.jpg'
 
+const SCROLL_OFFSET = 200;
```
Comment (abdullahmorrison) [inline src/pages/overlay/components/AmbassadorList.tsx:14]: put variable inside scrollListToAnimal function. it is only used in that function.

File: src/pages/overlay/components/AmbassadorList.tsx [hunk 2]
```diff
@@ -11,10 +11,29 @@ import AnimalButton from '../../../utils/global/animalButton/AnimalButton'
...
+
 export interface AmbassadorListProps{
     showAnimalList: boolean
     chatChosenAmbassador?: string
 }
+
+function scrollListToAnimal(listElement: HTMLDivElement | null, name: string) {
```
Comment (abdullahmorrison) [inline src/pages/overlay/components/AmbassadorList.tsx:16]: Move function to the inside of AmbassadorList function. Move to under the useEffect

Comment (abdullahmorrison) [inline src/pages/overlay/components/AmbassadorList.tsx:16]: convert function to arrow function

File: src/pages/overlay/components/AmbassadorList.tsx [hunk 3]
```diff
@@ -34,6 +29,11 @@ function scrollListToAnimal(listElement: HTMLDivElement | null, name: string) {
     }
 }
 
+export interface AmbassadorListProps{
+    showAnimalList: boolean
+    chatChosenAmbassador?: string
+}
+
```
Comment (abdullahmorrison) [inline src/pages/overlay/components/AmbassadorList.tsx:36]: remove new line in between AmbassadorListProps and AmbassadorList

**Review by abdullahmorrison — CHANGES_REQUESTED** [review]

File: src/pages/overlay/components/AmbassadorList.tsx (hunk 2 above)
Comment (pjeweb) [reply src/pages/overlay/components/AmbassadorList.tsx]: should be fixed

Comment (pjeweb) [reply src/pages/overlay/components/AmbassadorList.tsx]: should be fixed

File: src/pages/overlay/components/AmbassadorList.tsx (hunk 3 above)
Comment (pjeweb) [reply src/pages/overlay/components/AmbassadorList.tsx]: should be fixed

**Comment by pjeweb** [comment]
Tried to match your code style better in general, like no semicolons.

**Review by abdullahmorrison — APPROVED** [review]

**Pull request merged**